verticapy/machine_learning/vertica/base.py

```diff
@@ -1001,10 +1002,7 @@ def _compute_features_importance(self, tree_id: Optional[int] = None) -> None:
         )
         importance = self._format_vector(self.X, importance)
         if isinstance(tree_id, int) and (0 <= tree_id < self.n_estimators_):
-            if hasattr(self, "features_importance_trees_"):
-                self.features_importance_trees_[tree_id] = importance
```

Comment:
The pylint test was complaining that self.features_importance_trees_ is used before it is defined.